docs/content/advanced/backend/taxes/inclusive-pricing.md

```diff
@@ -0,0 +1,188 @@
+# Tax Inclusive Pricing
+
+In this document, you’ll learn how tax-inclusive pricing works in Medusa.
+
+:::note
+
+Tax Inclusive Pricing is currently in beta mode and guarded by a feature flag. To enable it, set the environment variable `MEDUSA_FF_TAX_INCLUSIVE_PRICING` to `true`, and [run migrations](../migrations.md#migrate-command).
+
+:::
+
+## Introduction
+
+Countries have different tax rates, even if they share the same currency. In these cases, using one tax-exclusive price for a currency that is used by multiple countries or regions can show different price to the customer based on the tax rate of each region.
+
+Using Tax Inclusive Pricing, you can just specify the price that should be displayed to the customer for a specific currency.
+
+Then, Medusa handles calculating the tax amount using the tax rate and the tax-inclusive price. This is managed in the backend and relayed to accounting and analytics tools.
+
+## How is Tax Inclusivity Defined?
+
+Tax inclusivity can be toggled for regions, currencies, price lists, and shipping options either during creation or while editing. This is represented by the boolean attribute `includes_tax` available in the entities `Region`, `Currency`, `PriceList`, and `ShippingOption`. By default, this attribute is set to `false`.
+
+If you want to enable or disable this attribute for any of these entities, you can use the create or edit/update endpoints related to these entities as shown in the [Admin API reference](https://docs.medusajs.com/api/admin/).
+
+The value set for these entities can affect whether line items and shipping methods are tax inclusive or not.
+
+### How is Tax Inclusivity Defined for Line Items?
+
+:::info
+
+When a product is added to the cart, a line item is created to represent that product in the cart.
+
+:::
```

Comment:
I think keeping this one (and the one below) helps those who don't necessarily know much about the architecture get a brief understanding what the terms mean, especially since we don't have any glossary/terminology page that explains what the different terms are